etc/bastion/bastion.conf.dist

```diff
@@ -355,7 +355,7 @@
 #
 # ttyrecFilenameFormat (string)
 #    DESC: Sets the filename format of the output files of ttyrec for a given session. Magic tokens are: &bastionname, &uniqid, &account, &ip, &port, &user (they'll be replaced by the corresponding values of the current session). Then, this string (automatically prepended with the correct folder) will be passed to ttyrec's -F parameter, which uses strftime() to expand it, so the usual character conversions will be done (%Y for the year, %H for the hour, etc., see man strftime). Note that in a addition to the usual strftime() conversion specifications, ttyrec also supports #usec#, to be replaced by the current microsecond value of the time.
-# DEFAULT: %Y-%m-%d.%H-%M-%S.#usec#.&uniqid.ttyrec
+# DEFAULT: %Y-%m-%d.%H-%M-%S.#usec#.&uniqid.&account.&user.&ip.&port.ttyrec
```

Comment:
On this one however, I think the `dist` version is better (it's more precise), so let's apply the same value to `configuration.inc`:
```
$C->{'ttyrecFilenameFormat'} = '%Y-%m-%d.%H-%M-%S.#usec#.&uniqid.&account.&user.&ip.&port.ttyrec' if (not $C->{'ttyrecFilenameFormat'});
```

(around line 185)

Reply:
Changed in 9b06db1461ba91932cff469a9169ad0f309405af